scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,243 @@
+        current_line = comment.get("line")
+        original_line = comment.get("original_line")
+
+        status_text = ""
+        line_to_display = None
+        # is_effectively_outdated is no longer needed with the new distinct flags
+
+        if current_pos is None:
+            status_text = STATUS_IRRELEVANT
+            line_to_display = original_line
+        elif original_line is not None and current_line != original_line:
+            status_text = STATUS_OLD
+            line_to_display = current_line
+        else:
+            status_text = STATUS_CURRENT
+            line_to_display = current_line
+
+        if line_to_display is None:
+            line_to_display = "N/A"
+
+        if status_text == STATUS_IRRELEVANT and not args.include_irrelevant:
+            continue
+        if status_text == STATUS_OLD and args.exclude_old:
+            continue
+
+        # Update latest timestamp (only for comments that will be printed)
+        if created_at_str:
+            try:
+                # GitHub ISO format "YYYY-MM-DDTHH:MM:SSZ"
+                # Python <3.11 fromisoformat needs "+00:00" not "Z"
+                if sys.version_info < (3, 11):
+                    dt_str = created_at_str.replace("Z", "+00:00")
+                else:
+                    dt_str = created_at_str
+                current_comment_dt = datetime.datetime.fromisoformat(dt_str)
+                if latest_created_at_obj is None or current_comment_dt > latest_created_at_obj:
+                    latest_created_at_obj = current_comment_dt
+            except ValueError:
+                sys.stderr.write(f"Warning: Could not parse timestamp: {created_at_str}\n")
+
+        # Get other comment details
+        user = comment.get("user", {}).get("login", "Unknown user")
+        path = comment.get("path", "N/A")
+        body = comment.get("body", "").strip()
+
+        if not body:
+            continue
+
+        diff_hunk = comment.get("diff_hunk")
+        html_url = comment.get("html_url", "N/A")
+        comment_id = comment.get("id")
+        in_reply_to_id = comment.get("in_reply_to_id")
+
+        print(f"## Comment by: **{user}** (ID: `{comment_id}`){f' (In Reply To: `{in_reply_to_id}`)' if in_reply_to_id else ''}\n")
+        if created_at_str:
+            print(f"*   **Timestamp**: `{created_at_str}`")
+        print(f"*   **Status**: `{status_text}`")
+        print(f"*   **File**: `{path}`")
+        print(f"*   **Line**: `{line_to_display}`") # Label changed from "Line in File Diff"
+        print(f"*   **URL**: <{html_url}>\n")
+
+        print("\n### Context:")
+        print("```") # Start of Markdown code block
+        if diff_hunk and diff_hunk.strip():
+            if args.context_lines == 0: # User wants the full hunk
+                print(diff_hunk)
+            else: # User wants N lines of context (args.context_lines > 0)
+                hunk_lines = diff_hunk.split('\n')
+                header_printed_this_time = False
```

Comment:
You don't need this entire part of the code. Instead, can't you just do:
if hunk_lines and hunk_lines[0].startswith("@@ "):
  print(hunk_lines[0])
  hunk_lines = hunk_lines[1:] # Remove the header from the hunk.
print "\n".join(hunk_lines[-lines_to_print_count:])